src/utils/string-test.js

```diff
@@ -192,4 +195,51 @@ describe('hash', async () => {
     const hash2 = await hash('string2');
     expect(hash1).to.not.equal(hash2);
   });
+
+  it('should throw if subtle crypto API is missing', async () => {
+    // Spy on console.warn method.
+    sandbox.stub(self.console, 'warn');
+
+    // Temporarily remove the crypto API.
```

Comment:
for completeness, should you also remove msCrypto? then we don't have to make changes if the test runs on IE 11

Reply:
Good call!